### PR TITLE
a fix to support running DbotpredictPhishingWords from playground

### PR DESCRIPTION
**Pull request by lperryp**
## Description
A fix to support running DbotpredictPhishingWords from playground, currently might cause an error

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by kirbles19** [comment]
@lperryp Done.

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def find_words_contain_tokens(positive_tokens, words_to_token_maps):
 def try_get_incident_field(field):
     value = ''
     incident = demisto.incident()
-    if 'CustomFields' in incident and field in incident['CustomFields']:
+    if 'CustomFields' in incident and incident['CustomFields'] is not None and field in incident['CustomFields']:
```
Comment (yguezpa) [inline Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:134]: Maybe you want to do:
if 'CustomFields' in incident and incident['CustomFields'] and field in incident['CustomFields']
So it won't be True if incident['CustomFields'] is []
Your choice

**Review by yguezpa — CHANGES_REQUESTED** [review]

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py (hunk 1 above)
Comment (lperryp) [reply Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:134]: you're right, but I think it's not too critical in this case,` incident['CustomFields']` is a dict or None

Comment (yguezpa) [reply Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:134]: and it cannot be an empty dict ?

Comment (lperryp) [reply Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:134]: then `field in incident['CustomFields']` will be false (which in any case need to be checked)

**Review by yguezpa — APPROVED** [review]

**Pull request merged**